_build/templates/default/sass/_utility.scss

```diff
@@ -212,3 +211,13 @@
     }
   }
 }
+
+@mixin textLink {
+    color: $colorSplash;
+    text-decoration: none;
+    border-bottom: 1px dotted scale-color($colorSplash, $lightness: 50%);
```

Comment:
`border` is an old hack. The only benefit is IE-compatibility.

https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-style
https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-color

Reply:
Good to know - for some reason that CSS 3 feature slipped by me!